packages/math-input/src/components/input/math-input.tsx

```diff
@@ -263,13 +228,10 @@ class MathInput extends React.Component<Props, State> {
         this._root.style.fontSize = `${fontSizePt}pt`;
     };
 
-    /** Gets and cache they bounds of the keypadElement */
+    /** Returns the current bounds of the keypadElement */
     _getKeypadBounds: () => any = () => {
```

Comment:
I wonder if this could be a good opportunity to type this method. Not sure if this right, but maybe:
```suggestion
    _getKeypadBounds(): DOMRect {
```